conf/layer.conf

```diff
@@ -29,6 +29,9 @@ AOS_MAIN_NODE_HOSTNAME ?= "main"
 
 AOS_UM_COMPONENT_PREFIX ?= ""
 
+AOS_IDENTIFIER_PLUGIN_NAME ?= "visidentifier"
```

Comment:
We don't need to populate identifier params with variables. It should be done in IAM config. Remove this commit.